# srv/CallStaff.srv

string requestor_id
---
bool success
string message
